README.adoc

```diff
@@ -116,21 +117,21 @@ You'll apply various annotations to the properties and declaration of a Java cla
 
 The [hotspot=class file=0]`JavaInfo` class is annotated with an `@Type` annotation. 
 The [hotspot=type file=0]`@Type("java")` annotation maps this class to define the `java` object type in GraphQL.
-The `java` object type will give information on the system's Java installation. 
+The `java` object type gives information on the system's Java installation. 
 
 The [hotspot=description file=0]`@Description` annotation gives a description to the `java` object type in GraphQL.
-This description is what will appear in the schema and the documentation. 
+This description is what appears in the schema and the documentation. 
 Descriptions aren't required, but it's good practice to include them. 
 
 The [hotspot=name file=0]`@Name` annotation maps the `vendor` property to the `vendorName` property of the `java` object type in GraphQL. 
 The [hotspot=name file=0]`@Name` annotation is used when mapping a property in a Java object to a property of a different name in the GraphQL object type.
-Without an [hotspot=name file=0]`@Name` annotation, the Java object property will be automatically 
+Without an [hotspot=name file=0]`@Name` annotation, the Java object property is automatically 
 mapped to a GraphQL object type property of the same name. 
 
 All data types in GraphQL are nullable by default.
 Non-nullable properties are annotated with the [hotspot=nonnull file=0]`@NonNull` annotation.
 The [hotspot=getVendor file=0]`getVendor()` and [hotspot=getVersion file=0]`getVersion()` getter functions 
-will automatically be mapped to retrieve their respective properties in GraphQL. 
+are automatically mapped to retrieve their respective properties in GraphQL. 
 If needed, setter functions are also supported and will be automatically mapped. 
```

Comment:
Change "will be automatically mapped" to "automatically mapped" to avoid future tense.